.github/workflows/docs.yaml

```diff
@@ -24,10 +36,10 @@ jobs:
           pip install -r docs/requirements.txt
       - name: Build docs for link check
         run: mkdocs build
-      - name: Link Checker
+      - name: Link Checker (generated site)
         id: lc
         uses: peter-evans/link-checker@v1
```

Comment:
Can you please add a not why this action is utilized here instead of the `dhbolach/[..]` link checker?